ge/org.osate.ge/src/org/osate/ge/internal/ui/handlers/AgeHandlerUtil.java

```diff
@@ -9,16 +9,18 @@
 import org.eclipse.jface.viewers.StructuredSelection;
 import org.eclipse.ui.IEditorPart;
 import org.eclipse.ui.ISources;
+import org.eclipse.ui.IWorkbenchWindow;
 import org.eclipse.ui.handlers.HandlerUtil;
 import org.osate.ge.internal.diagram.runtime.DiagramElement;
 import org.osate.ge.internal.services.UiService;
 import org.osate.ge.internal.ui.util.SelectionUtil;
+import org.osate.ge.internal.ui.util.UiUtil;
 
 class AgeHandlerUtil {
 	// Returns the current selection as diagram elements.
 	// If one or more of the selected objects cannot be adapted to DiagramElement then an empty list is returned.
 	public static List<DiagramElement> getSelectedDiagramElements(final ExecutionEvent event) {
-		final ISelection selection = HandlerUtil.getCurrentSelection(event);
+		final ISelection selection = getCurrentSelection();
```

Comment:
This method is now the same as getDiagramElementsFromCurrentSelection() and does not use the event parameter.

Reply:
@RyanMcilnay good catch. Should be fixed now.